packages/integrations/mdx/README.md

```diff
@@ -78,116 +78,98 @@ Visit the [MDX docs](https://mdxjs.com/docs/what-is-mdx/) to learn about using s
 
 Once the MDX integration is installed, no configuration is necessary to use `.mdx` files in your Astro project.
 
-You can extend how your MDX is rendered by adding remark, rehype and recma plugins.
+You can configure how your MDX is rendered with the following options:
 
-- [`extendPlugins`](#extendplugins)
-- [`remarkRehype`](#remarkrehype)
-- [`remarkPlugins`](#remarkplugins)
-- [`rehypePlugins`](#rehypeplugins)
+- [Markdown configuration options](#markdown-configuration-options)
+- [`extendMarkdownConfig`](#extendmarkdownconfig)
 - [`recmaPlugins`](#recmaplugins)
 
-### `extendPlugins`
+### Markdown configuration options
```

Comment:
nit: it feels weird to call these "Markdown configuration options"

I know this is shorthand for mirrors "all the stuff you can configure in Markdown, too!" but it really *is* MDX configuration options, that happen to (except for one :sweat_smile:) be the exact same options that are available elsewhere for configuring Markdown.

I might be tempted to still, in fact, call this MDX configuration options. I think that's what it *is*. And, I think if people are searching for "MDX configuration" then having that in a header is helpful for our Algolia search results.

Reply:
Hm, the overarching section is already MDX configuration, so I don't think that would make sense. I vote removing the heading if that's a concern 🤷